modules/python/test/test.py

```diff
@@ -167,4 +135,4 @@ def test_inheritance(self):
     NewOpenCVTests.repoPath = args.repo
     random.seed(0)
     unit_argv = [sys.argv[0]] + other;
-    unittest.main(argv=unit_argv)
+    unittest.main(argv=unit_argv)
```

Comment:
I do not see where all the other tests are imported? Our buildbot only runs "python test.py". How all the tests that you wrote are executed?